Cargo.toml

```diff
@@ -52,6 +56,7 @@ gix-tempfile = { version = "11.0.0", features = ["signals"] }
 hashbrown = "0.14.2"
 indicatif = "0.17.7"
 js-sys = "0.3.65"
+lazy_static = "1.4.0"
```

Comment:
We can use `once_cell` https://doc.rust-lang.org/std/cell/struct.OnceCell.html instead of `lazy_static`

Reply:
Sorry, I didn't leave a comment but I removed prior implementation, so once_cell is not applicable anymore.

Reply:
Ah ok ok, no problem. So can `lazy_static` dependency be removed from `Cargo.toml`?

Reply:
Yes. Thank you. I removed it.